packages/p2p/src/components/my-profile/payment-methods/payment-method-card/payment-method-card.jsx

```diff
@@ -119,4 +128,19 @@ const PaymentMethodCard = ({
     );
 };
 
+PaymentMethodCard.propTypes = {
+    add_payment_method: PropTypes.string,
+    is_add: PropTypes.bool,
+    is_vertical_ellipsis_visible: PropTypes.bool,
+    show_payment_method_name: PropTypes.bool,
+    label: PropTypes.string,
+    large: PropTypes.bool,
+    medium: PropTypes.bool,
+    onClick: PropTypes.func,
+    onClickAdd: PropTypes.func,
+    payment_method: PropTypes.object,
+    small: PropTypes.bool,
+    style: PropTypes.object,
+};
+
```

Comment:
PropTypes was not declared in this component, are the above added prop types fine with this component?

Reply:
O_O so there's a `small` `medium` `large` boolean?

Reply:
can't we use sizes? size = `small` | `medium` | `large`